nncf/parameters.py

```diff
@@ -118,3 +121,43 @@ def convert_ignored_scope_to_list(ignored_scope: Optional[IgnoredScope]) -> List
         raise RuntimeError('Legacy ignored scope format does not support '
                            'operation types')
     return results
+
+
+def get_ignored_node_names_from_ignored_scope(ignored_scope: IgnoredScope,
```

Comment:
What do you think about moving IgnoredScope and `get_ignored_node_names_from_ignored_scope` to the common/scopes.py?

Reply:
Done